gengokit/generator/gen_test.go

```diff
@@ -29,6 +28,7 @@ func init() {
 
 func init() {
 	log.SetLevel(log.DebugLevel)
+	_ = errors.Wrap
```

Comment:
If this is the only place that `errors` is referenced, we should just remove the import. If this isn't the only place that `errors` is referenced, we should remove this line.